tokio/src/sync/mpsc/bounded.rs

```diff
@@ -1036,6 +1036,37 @@ impl<T> Sender<T> {
             chan: self.chan.downgrade(),
         }
     }
+
+    /// Returns the max buffer capacity of the channel.
+    ///
+    /// The max capacity is the buffer capacity initially specified when calling
+    /// [`channel`].
```

Comment:
maybe we should also discuss how this is distinct from `capacity` here in the docs, as well as in the examples?

perhaps something like this:
```suggestion
    /// The max capacity is the buffer capacity initially specified when calling
    /// [`channel`]. This is distinct from [`capacity`], which returns the *current*
    /// available buffer capacity: as messages are sent and received, the
    /// value returned by [`capacity`] will go up or down, while  `max_capacity`
    /// will always return maximum capacity that was set when the channel was
    /// created.
```

Reply:
it might also be worth adding something similar to the docs for `capacity`, referencing this method?

Reply:
SGTM